src/PowerShellEditorServices/Services/DebugAdapter/DebugService.cs

```diff
@@ -753,56 +661,24 @@ public VariableScope[] GetVariableScopes(int stackFrameId)
             };
         }
 
-        /// <summary>
-        /// Clears all breakpoints in the current session.
-        /// </summary>
-        public async Task ClearAllBreakpointsAsync()
-        {
-            try
-            {
-                PSCommand psCommand = new PSCommand();
-                psCommand.AddCommand(@"Microsoft.PowerShell.Utility\Get-PSBreakpoint");
-                psCommand.AddCommand(@"Microsoft.PowerShell.Utility\Remove-PSBreakpoint");
-
-                await this.powerShellContext.ExecuteCommandAsync<object>(psCommand).ConfigureAwait(false);
-            }
-            catch (Exception e)
-            {
-                logger.LogException("Caught exception while clearing breakpoints from session", e);
-            }
-        }
-
         #endregion
 
         #region Private Methods
 
         private async Task ClearBreakpointsInFileAsync(ScriptFile scriptFile)
         {
             // Get the list of breakpoints for this file
-            if (this.breakpointsPerFile.TryGetValue(scriptFile.Id, out List<Breakpoint> breakpoints))
-            {
-                if (breakpoints.Count > 0)
-                {
-                    PSCommand psCommand = new PSCommand();
-                    psCommand.AddCommand(@"Microsoft.PowerShell.Utility\Remove-PSBreakpoint");
-                    psCommand.AddParameter("Id", breakpoints.Select(b => b.Id).ToArray());
-
-                    await this.powerShellContext.ExecuteCommandAsync<object>(psCommand).ConfigureAwait(false);
+            // if (_breakpointService.BreakpointsPerFile.TryGetValue(scriptFile.Id, out HashSet<Breakpoint> breakpoints))
+            // {
+                // if (breakpoints.Count > 0)
+                // {
+                    await _breakpointService.RemoveBreakpointsAsync(_breakpointService.GetBreakpoints()
+                        .Where(bp => bp is LineBreakpoint lbp && string.Equals(lbp.Script, scriptFile.FilePath))).ConfigureAwait(false);
 
                     // Clear the existing breakpoints list for the file
-                    breakpoints.Clear();
-                }
-            }
-        }
-
-        private async Task ClearCommandBreakpointsAsync()
-        {
-            PSCommand psCommand = new PSCommand();
-            psCommand.AddCommand(@"Microsoft.PowerShell.Utility\Get-PSBreakpoint");
-            psCommand.AddParameter("Type", "Command");
-            psCommand.AddCommand(@"Microsoft.PowerShell.Utility\Remove-PSBreakpoint");
-
-            await this.powerShellContext.ExecuteCommandAsync<object>(psCommand).ConfigureAwait(false);
+                    // breakpoints.Clear();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/TylerLeonhardt/PowerShellEditorServices/pullRequest?prid=4598462)